validator/project_validator.go

```diff
@@ -1374,33 +1374,73 @@ func validateProjectTaskIdsAndTags(project *model.Project) ValidationErrors {
 func checkTaskRuns(project *model.Project) ValidationErrors {
 	var errs ValidationErrors
 	for _, bvtu := range project.FindAllBuildVariantTasks() {
+		if len(bvtu.AllowedRequesters) != 0 {
+			for _, requester := range bvtu.AllowedRequesters {
+				if !utility.StringSliceContains(evergreen.AllRequesterTypes, requester) {
+					errs = append(errs, ValidationError{
+						Level: Error,
```

Comment:
In the work to make project validation more consistent, validators were split by warning/error (the names here should maybe include the word warning to make that clearer, but checkTaskRuns is in the projectWarningValidators). We don't check warning validators everywhere we check error ones, I believe. This error may best live in [validateBVFields](https://github.com/evergreen-ci/evergreen/blob/96b3f970f8e436d639cc8ee19456d225dda3384d/validator/project_validator.go#L661)

Reply:
Hmm, I just removed this validation since it's a duplicate. The requesters already get checked for valid values when translating the parser project to the project.